content/neuvector-prime/azure/contents.lr

```diff
@@ -190,53 +199,46 @@ Billing will be available in Azure Portal billing
 Home > Cost Management <subscription> | Cost analysis
 
 
+## Enable Auto Upgrade minor version
 
-## Upgrading a NeuVector PAYG Cluster
+By default the Kubernetes Service Extension "Auto Upgrade minor version" will be in a Disabled state.
 
-The Azure marketplace PAYG offer is periodically updated as a new versions of NeuVector are released, and for optimizations in the integration with Azure. To update the latest version of the NeuVector PAYG offering supported in the marketplace listing, use the "az k8s-extension update" command.
+To enable Auto upgrade minor version
 
-As a Prerequisites to create extension instances in AKS cluster,  Register Azure Feature
+1. Click Extensions name
+2. Check Auto upgrade minor version 
 
-Execute the following command in Cluster Cloud Shell:
+This will take few seconds to update. In the Notifications, "Updating extension" job will be in running state. 
```

Comment:
This needs more  detail. When you first enable it , it will show updating but that just means updating from non-auto to auto to vice versa depending on the current state and previous state.  Once it is enabled and it detects an update is available it will show "Updating" again but this time you will see the version has changed.  If they enable auto upgrade and they are on the latest, the would not see this second updating.

Reply:
With the latest decision, auto update is enabled by default. Section is removed.